### ISSUE TITLE
Selectors removed from media query optimization

### ISSUE THREAD

**Issue by danrichman** [jakubpawlowicz/clean-css#880]
Selectors aren't being retained in media query, in the following example. Just tested the following with `v4.0.4` directly in the [web interface](https://jakubpawlowicz.github.io/clean-css/) using default options. I also noticed the same problem with the latest gulp-clean-css `v2.3.2`.

### Input CSS
```css
@media(max-width:950px) {
    .body-format>p,
    .body-format>blockquote,
    .body-format>q,
    .body-format>ul,
    .body-format>ol,
    .body-format>h2,
    .body-format>h3 {
        left: 0;
        width: 90vw
    }
}
```

### Erroneous Output
```css
@media(max-width:950px){width:90vw}
```
Notice selectors have been removed and makes the CSS both invalid and non-usable.

**Comment by jakubpawlowicz** [comment]
That's because of lack of space after `@media` - fix is coming!

**Comment by jakubpawlowicz** [comment]
Fixed in 4.0.5.
